tasks/eva_2373/covid19dp_analysis.Rmd

```diff
@@ -0,0 +1,182 @@
+
+
+When looking over a log scale, we can see that there are variant calls that have AF between 0.2 and 0.8 `r nb_of_variant_call_gt0.2lt0.8` or `r format(round(nb_of_variant_call_gt0.2lt0.8/nb_of_variant_call*100, 1), nsmall = 1)`% 
+of the calls
+
+```{r echo=FALSE, results="hide", message=FALSE}
+ggplot(data, aes(AF)) + geom_histogram( binwidth = 0.01) + xlab("Allele frequency") + ylab("Number of variant call") + scale_y_log10() + facet_wrap(~variantType)
+```
+
+## Highest allele frequency
+
+Because a variant might have low AF in one Sample and a higher one in another, It makes sense to look at the highest AF per variant across all samples as a single high AF would justify to accession the variant. 
+
+```{r echo=FALSE, results="hide", message=FALSE}
+ggplot(data_per_variant, aes(max_AF)) + geom_histogram(binwidth = 0.01) + xlab("Max Allele frequency") + ylab("Number of variant") + facet_wrap(~variantType)
+```
+The distribution looks similar to the previous one and confirms that some of the variant have very low AF across all the samples where they were detected.
+
+## Variant frequency across samples
+
+Another way of assessing the robustness of the calls is to check if they are found in 1 or many samples. The goal here is to distinguish between low and high occurring variants so I grouped the number of occurrence in bin of variable size (1, 2, 3, 4-40, 41-53202). I also created groups of allele frequencies using the maximum occurring allele frequency shown above (low AF:0-0.2, mid AF: 0.2-0.8, high AF: 0.8-1.0). 
+
+```{r echo=FALSE, results="hide", message=FALSE}
+ggplot(data_per_variant, aes(x=binned_occurence)) + geom_bar() + xlab("Number of samples") + ylab("Number of variants") + facet_grid(max_AF_cat ~ variantType)
+```
+
+The mid and high AF have the same pattern for SNPs and INDELs and the variant seems to be occuring relatively frequently. However in the low AF category the SNPs and INDELs have different patterns. The INDELs have a large number of variants with low AF that appear in only a few samples. There is also a group ov variant (both SNPs and INDELs) that appear in a large number of sample.
+
+## Frequent variants with low allele frequencies
+
+```{r echo=FALSE, results="hide", message=FALSE}
+data_per_variant[data_per_variant$variantType=="INDEL" & data_per_variant$max_AF < 0.01 & data_per_variant$total>1000,]
+
+```
+
+
+## Distribution per sample
+
+```{r echo=FALSE, results="hide", message=FALSE}
+
+ggplot(data_per_sample, aes(total)) + geom_histogram(binwidth = 10) + xlab("Number of sample") + ylab("Number of variant") + facet_wrap( ~ variantType, scale="free_y")
```

Comment:
just a reminder about the axis labelling issue we discussed!

Reply:
Done